contrib/brl/b3p/expat/xmltok.h

```diff
@@ -80,20 +80,20 @@ extern "C" {
 #define XML_TOK_PREFIXED_NAME 41
 
 #ifdef XML_DTD
-#define XML_TOK_IGNORE_SECT 42
+#  define XML_TOK_IGNORE_SECT 42
```

Comment:
This is a style choice.  It can be turned off.  Many places  would indent the contents of if/else pragma statements.

Some code had it each way.  I went with indenting to make it easier to identify code blocks of pramas.